<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryFileList</class>
 <widget class="QWidget" name="HistoryFileList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>763</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>历史文件</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
            background-color: #F0F8FF; 
        }
        QPushButton {
            background-color: #E6F3FF;
            border: 1px solid #B0C4DE;
            border-radius: 4px;
            padding: 5px 15px;
            color: #2F4F4F;
            font-size: 16pt;
        }
        QPushButton:hover {
            background-color: #CCE8FF;
        }
        QPushButton:pressed {
            background-color: #B0D8FF;
        }

    </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>时间：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="date_Start">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QDateTimeEdit {
    background-color: #E6F3FF;  /* 设置背景为淡蓝色 */
    border: 1px solid #A3C8FF;  /* 设置边框为蓝色 */
    border-radius: 5px;         /* 圆角边框 */
    padding: 5px;               /* 设置内边距 */
    color: #333;                /* 设置文本颜色为深灰色 */
 padding-right: 0px; 
}


QDateTimeEdit::up-button {
    image: url(:/up.png);  /* 设置自定义图标路径 */
    border: none;               /* 去掉默认边框 */
    border-radius: 5px;         /* 圆角 */
    width: 30px;                /* 设置按钮宽度 */
    height: 30px;               /* 设置按钮高度 */
    padding: 0px;               /* 去掉按钮内部的内边距 */
    margin-right: 0px;          /* 去掉按钮右边的外边距 */
}
QDateTimeEdit::down-button {
    image: url(:/down.png);  /* 设置自定义图标路径 */
 border: none;               /* 去掉默认边框 */
    border-radius: 5px;         /* 圆角 */
    width: 30px;                /* 设置按钮宽度 */
    height: 30px;               /* 设置按钮高度 */
    padding: 0px;               /* 去掉按钮内部的内边距 */
    margin-right: 0px;          /* 去掉按钮右边的外边距 */
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>~</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="date_End">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QDateTimeEdit {
    background-color: #E6F3FF;  /* 设置背景为淡蓝色 */
    border: 1px solid #A3C8FF;  /* 设置边框为蓝色 */
    border-radius: 5px;         /* 圆角边框 */
    padding: 5px;               /* 设置内边距 */
    color: #333;                /* 设置文本颜色为深灰色 */
 padding-right: 0px; 
}


QDateTimeEdit::up-button {
    image: url(:/up.png);  /* 设置自定义图标路径 */
    border: none;               /* 去掉默认边框 */
    border-radius: 5px;         /* 圆角 */
    width: 30px;                /* 设置按钮宽度 */
    height: 30px;               /* 设置按钮高度 */
    padding: 0px;               /* 去掉按钮内部的内边距 */
    margin-right: 0px;          /* 去掉按钮右边的外边距 */
}
QDateTimeEdit::down-button {
    image: url(:/down.png);  /* 设置自定义图标路径 */
 border: none;               /* 去掉默认边框 */
    border-radius: 5px;         /* 圆角 */
    width: 30px;                /* 设置按钮宽度 */
    height: 30px;               /* 设置按钮高度 */
    padding: 0px;               /* 去掉按钮内部的内边距 */
    margin-right: 0px;          /* 去掉按钮右边的外边距 */
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
    background-color: #E6F3FF;
    border: 1px solid #B0C4DE;
    border-radius: 4px;
    padding: 5px 15px;
    color: #2F4F4F;
    font-size: 16pt;
}

QComboBox:hover {
    background-color: #CCE8FF;
}

QComboBox:pressed {
    background-color: #B0D8FF;
}

QComboBox::drop-down {
    border: none;
    width: 15px;
}

QComboBox::down-arrow {
   image: url(:/arrow.png);  /* 替换成你的图片路径 */
    width: 22px;
    height: 22px;
}

QComboBox QAbstractItemView {
    background-color: #F0F8FF;
    border: 1px solid #B0C4DE;
    selection-background-color: #CCE8FF;
    selection-color: #2F4F4F;
}</string>
          </property>
          <item>
           <property name="text">
            <string>警报图片</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>原始数据</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>关    闭</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>删除文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_all">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>全部删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>复    检</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
